docs/examples/model_ex/model_examples_pull.py

```diff
@@ -2,7 +2,7 @@
 Pull plot
```

Comment:
Data vs Model, pull comparison

Reply:
Since we are already in the data vs model section, I prefer to keep it simpler.